go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/google/go-jsonnet
+
+require (
+	github.com/fatih/color v1.7.0
```

Comment:
At some point we got complaints that we depend on `color` even for people who use go-jsonnet as a library only. Is there a way to have separate dependencies for the library and the command?

Reply:
ok, it's good then